src/databricks/sql/cloudfetch/downloader.py

```diff
@@ -171,3 +202,34 @@ def decompress_data(compressed_data: bytes) -> bytes:
                 uncompressed_data += data
                 start += num_bytes
         return uncompressed_data
+
+
+def http_get_with_retry(url, max_retries=5, backoff_factor=2, download_timeout=60):
```

Comment:
In here it does not bother reading retry-after header? Or there is none because this is directly getting from cloud files?

Reply:
this is not thrift. Just cloud files.